test/typescript/providers/main.ts

```diff
@@ -59,14 +59,21 @@ export class NamespacedProviders extends TerraformStack {
       "callerIdentity",
       {}
     );
+
+    const role = new Aws.iam.IamRole(this, "role", {
+      assumeRolePolicy: "assumeRolePolicy",
+    });
+    new Aws.iam.IamRolePolicyAttachment(this, "lambda-role-vpc-att", {
+      policyArn:
+        "arn:aws:iam::aws:policy/service-role/AWSLambdaVPCAccessExecutionRole",
+      role: role.name,
```

Comment:
that's the one which changed before, right? Without these changes it would have to be `role.name!`?

Reply:
exactly 👍

Reply:
top 👍